operations/messages.md

```diff
@@ -0,0 +1,149 @@
+
+#### Sender
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Discriminator` | string [Sender discriminator](#sender-discriminator) | required | If the message was created by enterprise or integration. |
+
+#### Sender discriminator
+
+* `Enterprise`
+* `Integration`
+
+## Add message
+
+Adds a message to the [Message thread](messagethreads.md#message-thread) that was created by the integration.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/messages/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "Messages": [
+        {
+            "Text": "Text of the message",
+            "MessageThreadId": "8d02142f-31cf-4115-90bf-ae5200c7a1ba",
+            "Identifier": "Message 1"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Messages` | array of [Message](#added-message)s | required | Message to be added to the given [Message thread](messagethreads.md#message-thread). |
+
+### Response
+
+```javascript
+{
+    "Messages": [
+        {
+            "Identifier": "Message 1",
+            "Message": {
+                "Id": "4bfe4675-98a9-4428-9ea9-ae5300dbaa93",
+                "ThreadId": "8d02142f-31cf-4115-90bf-ae5200c7a1ba",
+                "Text": "Text of the message",
+                "Sender": {
+                    "Discriminator": "Integration",
+                    "Value": null
+                },
+                "CreatedUtc": "2022-03-09T13:19:46Z"
+            }
+        }
+    ]
+}
+``` 
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Messages` | array of [Added message](#added-messages)s | required | Collection of created messages. |
+
+#### Added Message
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Identifier` | string | optional | Identifier of the created message. |
+| `Message` | string | [Message](#message) | Message that was added to the [Message thread](messagethreads.md#message-thread). |
+
+
+#### Message parameters
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Text` | string | required | Text of the message. |
+| `ThreadId` | string | required | Unique identifier of the [Message thread](messagethreads.md#message-thread) to which the message should be added. |
+| `Identifier` | string | optional | Identifier of the message to be created. |

```

Comment:
Add a general guidelines section about Identifiers, what they are and how to use them. Done as be next PR.